samples/app-cache-meetings/nodejs/src/components/app-cache-tab.tsx

```diff
@@ -0,0 +1,99 @@
+// <copyright file="app-cache-tab.tsx" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// </copyright>
+
+import { useState } from "react";
+import React from "react";
+import * as microsoftTeams from "@microsoft/teams-js";
+
+/// </summary>
+/// function logitem
+/// </summary>
+function logItem(action: string, actionColor: string, message: string) {
+    const newItem =
+        "<span style='font-weight:bold;color:" +
+        actionColor +
+        "'>" +
+        action +
+        "</span> " +
+        message +
+        "</br>";
+    return newItem;
+}
+
+/// </summary>
+/// In beforeUnloadHandler using setItems and readyToUnload callback function
+/// </summary>
+const beforeUnloadHandler = (
+    readyToUnload: () => void) => {
+    readyToUnload();
+    return true;
+};
+
+/// </summary>
+/// loadHandler using setItems to set values
+/// </summary>
+const loadHandler = (
+    setItems: React.Dispatch<React.SetStateAction<string[]>>,
+    data: microsoftTeams.LoadContext) => {
+    logItem("OnLoad", "blue", "Started for " + data.entityId);
+
+    let newItem = logItem("OnLoad", "blue", "Completed for " + data.entityId);
+    setItems((Items) => [...Items, newItem]);
+
+    microsoftTeams.app.notifySuccess();
+};
+
+const AppCacheTab = () => {
+    const [items, setItems] = useState<string[]>([]);
+    const [title] = useState("App Cache Testing Sample");
+    const [initState] = useState(true);
+
+    React.useEffect(() => {
+        if (!initState) {
+            return;
+        }
+
+        microsoftTeams.app.initialize().then(() => {
+
```

Comment:
get the context and make the entity id the title of the Tab. Will help identify the order of events.

Reply:
Comment Resolved : 
<img width="349" alt="image" src="https://user-images.githubusercontent.com/109147706/211606229-4d78ee65-b922-447e-85d4-9e675b775581.png">

Reply:
Update all the screenshots/gifs